mesheryctl/internal/cli/root/adapter/adapter.go

```diff
@@ -87,6 +87,7 @@ var (
 			// ensure the mesh's adapter is available and update adapterURL if so
 			if err = validateAdapter(mctlCfg, meshName); err != nil {
 				utils.Log.Error(err)
+				return err
 			}
```

Comment:
n the original code, even if the required adapter is not found, the process continues and results in a misleading success message. To address this, I've modified the logic so that when an adapter is missing, an error is returned immediately, which stops further execution. This ensures that deployment does not proceed when the necessary adapter isn’t available, making the logs clearer and preventing false-positive.